modules/Emails/EmailsController.php

```diff
@@ -449,6 +450,24 @@ public function action_GetFromFields()
             }
         }
 
+        if ($sugar_config['email_allow_send_as_user']) {
+            $data[] = array(
+                'type' => 'personal',
+                'id' => $current_user->id,
+                'attributes' => array(
+                    'from' => $current_user->email1,
+                    'name' => $current_user->full_name,
+                ),
+                'prepend' => $prependSignature,
+                'isPersonalEmailAccount' => true,
+                'isGroupEmailAccount' => false,
+                'emailSignatures' => array(
+                    'html' => utf8_encode(html_entity_decode($defaultEmailSignature['signature_html'])),
```

Comment:
`$defaultEmailSignature['signature_html']` could be unset. can you please check if it's exists? (see above)

Reply:
I'll correct this.